src/Mailer/Mailer.php

```diff
@@ -129,7 +130,7 @@
  * @method array|string getBody(?string $type = null) Get generated message body as array.
  *   {@see \Cake\Mailer\Message::getBody()}
  */
-#[\AllowDynamicProperties]
+#[AllowDynamicProperties]
```

Comment:
Do we have to do this? I find the FQCN inline cleaner here for attributes which are to some extend meta info, even if not on the same level as docblocks.

Reply:
I could find that out :) We are using the same ruleset for our spryker repos using \FQCN